File: app/src/main/java/net/squanchy/service/firebase/model/FirebaseSpeaker.java
package net.squanchy.service.firebase.model;

import android.support.annotation.NonNull;
import android.support.annotation.Nullable;

public class FirebaseSpeaker {

    @NonNull
    public String id;

    @NonNull
    public String name;

    @NonNull
    public String bio;

    @Nullable
    public String company_name;

    @Nullable
    public String company_url;

    @Nullable
    public String personal_url;

    @Nullable
    public String twitter_username;

    @Nullable
    public String photo_url;
}

File: app/src/main/java/net/squanchy/service/firebase/model/FirebaseSpeaker.kt
package net.squanchy.service.firebase.model

data class FirebaseSpeaker(
    var id: String? = null,
    var name: String? = null,
    var bio: String? = null,
    var company_name: String? = null,
    var company_url: String? = null,
    var personal_url: String? = null,
    var twitter_username: String? = null,
    var photo_url: String? = null
)
